app/code/community/Oyst/OneClick/controllers/Adminhtml/Oneclick/ActionsController.php

```diff
@@ -77,4 +77,74 @@ public function indexAction()
     {
         $this->_initAction()->renderLayout();
     }
+
+    /**
+     * Download requested log file
+     */
+    public function downloadAction()
+    {
+        $params = Mage::app()->getRequest()->getParams();
+        $fileName = Mage::getBaseDir('var') . DS . 'log' . DS . $params['name'];
+
+        $this->missingFile($fileName, $params['name']);
+
+        if (file_exists($fileName) && is_readable($fileName)) {
```

Comment:
Some [stuff to fix](https://travis-ci.org/oystparis/oyst-1click-magento/jobs/335081866#L1243)

Exemple : https://magento.stackexchange.com/a/174668/56191